scribble-lib/scribble/acmart/lang.rkt

```diff
@@ -22,12 +22,13 @@
            [natbib? #f]
            [anonymous? #f]
            [authorversion? #f]
-           [font-size #f])
+           [font-size #f]
+           [nonacm? #f])
```

Comment:
Its not a boolean-valued variable, so change the name to `nonacm` ?

Reply:
I would, but in this case I thought it better to follow the convention of the other variables. So maybe we could remove the `?` on all of them?

Reply:
Oh I didn't notice them --- nevermind